### ISSUE TITLE
A question about mixup3D

### ISSUE THREAD

**Issue by QiuDeZhang** [PuFanqi23/MonoDGP#5]
**Thanks for your excellent work.**

I have a question about mixup3D:

When I submit the kitti test without any modification, I can achieve the results claimed in the paper：

<img width="298" alt="Image" src="https://github.com/user-attachments/assets/1b4938aa-fb20-438a-b939-b1bce1e6c195" />

However, when I remove mixup3D (as far as I know, it was first proposed by MonoLSS), the results on the test set dropped significantly, even worse than the original MonoDETR (25.00, 16.47, 13.58). 

<img width="286" alt="Image" src="https://github.com/user-attachments/assets/f99d2d06-040a-44ff-97b7-6ff26c078280" />

<img width="290" alt="Image" src="https://github.com/user-attachments/assets/44e3f835-7bfb-4430-8004-a83b9d9812db" />

I wonder if mixup3D plays an important role?

Thanks for your reply.

**Comment by PuFanqi23** [comment]
Hi, thanks for your interest in MonoDGP.

Mixup3D can improve the performance, but the effect won't be such significant. It looks like you didn't fully remove the Mixup3D. For example, check out whether you have blended image A and B, but forget to add B labels. Or the other way around, you have commented out the blending code, but still add B labels to image A.

If you still encounter this problem, try following steps to further evaluate the Mixup3D method:

1. Set random_mixup3d to 0 in configs/monodgp.yaml. This is a simple way to disable Mixup3D.

2. We recommend directly applying Mixup3D to original MonoDETR. It 's easy to transfer as we have a generic dataloader code.

You can firstly train models without Mixup3D on KITTI val set. If the performance gap on the val set is still 3%~4% instead of 0.6%~1.0%,  please feel free to contact me for further discussion.

**Comment by QiuDeZhang** [comment]
Hi, thanks for your reply.
I have tried to follow the first step to further evaluate the Mixup3D method

![Image](https://github.com/user-attachments/assets/e7954453-c42c-4daa-ab3c-1e70eb8b9c2f)

and get results as follow.

![Image](https://github.com/user-attachments/assets/1598abdf-defb-4c17-9ad9-1d3b44e36848)

This problem seems to still exist.

**Comment by PuFanqi23** [comment]
Hi. Have you modified kitti_dataset.py? Is the model using Mixup3D performs normal on val set? 

I create a new environment and reproduce the result without Mixup3D after you proposed this issue, but my moderate AP is 21.03 as follows:

![Image](https://github.com/user-attachments/assets/78b8b882-27ce-40b4-ae83-9e6207edc088)

Applying Mixup3D to other methods like MonoLSS, MonoCon, MonoDETR can't improve up to more than 3%. Here is the MonoLSS ablation study:

![Image](https://github.com/user-attachments/assets/02aaf930-855f-44d2-a12b-ba47b9274f0b)

Hope you can test this trick on other methods.

**Comment by PuFanqi23** [comment]
BTW,  our reproduced results on the baseline model are often 1~2% lower than the paper reported, so we have utilized these tricks to narrow this performance gap.

**Comment by QiuDeZhang** [comment]
Hi, This is a result w/ Mixup3D on our device.
![Image](https://github.com/user-attachments/assets/e9ca8ca8-0c50-45e2-87f9-869910410a2f)

w/ Mixup3D (21.48) v.s. w/o Mixup3D (20.13) 

The performance gap seems to be relatively small.

Thank you for your prompt and rigorous response！
